.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "typescript.tsdk": "node_modules/typescript/lib"
```

Comment:
What is this TchonaTchonis ? 
It should comes on push ?
maybe you set it as GitIgnore for personal archives.